### PR TITLE
Cython again

### PR DESCRIPTION
**Pull request by wkerzendorf**
@unoebauer now at 0.29

### CODE REVIEW

**Comment by unoebauer** [comment]
Travis does not seem to be happy with the new version...

**Comment by wkerzendorf** [comment]
@unoebauer - so I've now seen this issue pop up in the wild. it seems running it a second time does not result in a crash.

**Comment by unoebauer** [comment]
so what do we do about this? Travis is still failing.

File: ci-helpers/install_tardis_env.sh [hunk 1]
```diff
@@ -8,12 +8,12 @@ else
...
-   git clone https://github.com/cython/cython
-   cd cython
-   git checkout c485b1b77264c3c75d090a3c526de24966830d42
-   CFLAGS="$CFLAGS -D CYTHON_CLINE_IN_TRACEBACK=0" python setup.py install
-   cd ..
+   #conda uninstall -y cython
+   #git clone https://github.com/cython/cython
+   #cd cython
+   #git checkout c485b1b77264c3c75d090a3c526de24966830d42
+   #CFLAGS="$CFLAGS -D CYTHON_CLINE_IN_TRACEBACK=0" python setup.py install
```
Comment (unoebauer) [inline ci-helpers/install_tardis_env.sh]: I think we should just leave this workaraound in.

File: tardis_env27.yml [hunk 2]
```diff
@@ -13,7 +13,7 @@ dependencies:
 - matplotlib=2.0
 - astropy=1.3
 - numexpr=2.6
-- Cython=0.28.4
+- Cython=0.29
```
Comment (unoebauer) [inline tardis_env27.yml]: Let's test offline whether this version works. We could add a "troubleshooting" section to the docu about cython issues and our workaround, namely compiling cython by hand.

**Comment by unoebauer** [comment]
Well, I would partly abandon hope and just stick with our workaround:
- keep cython in the conda env file
- in Travis, uninstall the conda cython and compile cython by hand
- add a troubleshooting section to the docu about cython issues, particularly what to do when encountering segfaults.

**Comment by wkerzendorf** [comment]
@unoebauer the switch to python 3 cleaned this issue up

**Pull request closed without merging**